src/auth.js

```diff
@@ -343,7 +366,11 @@ export default class DropboxAuth {
    */
   authenticateWithCordova(successCallback, errorCallback) {
     const redirectUrl = 'https://www.dropbox.com/1/oauth2/redirect_receiver';
-    const url = this.getAuthenticationUrl(redirectUrl);
```

Comment:
This wouldn't work here, since the promise wouldn't resolve before the rest of the code executes